sorc/link_workflow.sh

```diff
@@ -131,6 +131,10 @@ fi
 #---------------------------------------
 #--add files from external repositories
 #---------------------------------------
+#--copy/link NoahMp table form ccpp-physics repository
+cd "${HOMEgfs}/parm/ufs" || exit 1
+${LINK_OR_COPY} "${HOMEgfs}/sorc/ufs_model.fd/FV3/ccpp/physics/physics/noahmptable.tbl" .
```

Comment:
Forgot to mention, but this file needs to be added to the list of ignored files in `.gitignore`